lib/utils/tracker.js

```diff
@@ -7,8 +7,12 @@
 const ga = require('react-native-google-analytics-bridge');
 const { analyticsTrackerId } = require('../../config');
```

Comment:
You are getting the entire `config` object on line 13. I guess we can save one `require` call.